Unicode Delphi has incorrect `TestExtendedInterfaceRTTI` output, but …

hvassbotn opened the pull request:
…non-Unicode Delphi has correct output #14

https://github.com/jpluimers/HVRTTIUtils/issues/14

jpluimers committed:
New unit tests InterfaceVariantsTestsUnit based on TestInterfaceVariants.dpr

jpluimers committed:
New unit tests in PolymorphicPublishedMethodsTestsUnit based on TestPolyPub.dpr

jpluimers committed:
Cleanup and referencing location of original test projects from the unit tests.

Commit:
Unicode Delphi has incorrect `TestExtendedInterfaceRTTI` output, but non-Unicode Delphi has correct output #14
https://github.com/jpluimers/HVRTTIUtils/issues/14

jpluimers commented:
This line now fails to compile: https://github.com/hvassbotn/HVRTTIUtils/blob/develop/HVMethodInfoClasses.pas#L53

The reason is that this method is now missing from https://github.com/hvassbotn/HVRTTIUtils/blob/develop/HVMethodSignature.pas:

```
function Skip(const Value: PSymbolName): Pointer;
```

That change was in https://github.com/jpluimers/HVRTTIUtils/pull/15/commits/6b51850cc793f817d7ae9b76365497189584aee9

Can you make sure all sample applications still compile in this pull request?

Commit:
Revert "New unit tests InterfaceVariantsTestsUnit based on TestInterfaceVariants.dpr"

This reverts commit f2badf16e540927f050c5aa69a3237ff9f77c699.

Conflicts:
	Tests/HVRTTIUtilsTests.dpr

Commit:
Revert "New unit tests in PolymorphicPublishedMethodsTestsUnit based on TestPolyPub.dpr"

This reverts commit a2bd36d3e6765a8712599f9def11c532e933b359.

Conflicts:
	Tests/HVRTTIUtilsTests.dpr

Commit:
Revert "Cleanup and referencing location of original test projects from the unit tests."

This reverts commit c4a1467a7da830ded2ae0b4c9dfeff8e3a4b8d21.

Commit:
Make all projects compile:
Unicode Delphi has incorrect `TestExtendedInterfaceRTTI` output, but non-Unicode Delphi has correct output #14
https://github.com/jpluimers/HVRTTIUtils/issues/14

hvassbotn commented:
sorry about that - pushed fixes now so that all should compile
My commit history is a bit of a mess - I tried to integrate changes from you before PR had been integrated - had to revert those
